### PR TITLE
Fix the progressive mp4 transcoding profile

### PR DESCRIPTION
**Pull request by nyanmisaka**
**Please add the "stable backport" tag, thanks!**

**Changes**
- Fix the progressive mp4 transcoding profile
mp4 should be used instead of webm for the maximum compatibility.

**Issues**
Fixes: https://github.com/jellyfin/jellyfin/issues/8089

### CODE REVIEW

File: src/scripts/browserDeviceProfile.js [hunk 1]
```diff
@@ -719,17 +718,15 @@ import browser from './browser';
...
+                Container: 'mp4',
                 Type: 'Video',
-                AudioCodec: webmAudioCodecs.join(','),
-                // TODO: Remove workaround when servers migrate away from 'vpx' for transcoding profiles.
-                VideoCodec: (canPlayVp8 ? webmVideoCodecs.concat('vpx') : webmVideoCodecs).join(','),
+                AudioCodec: videoAudioCodecs.join(','),
+                VideoCodec: mp4VideoCodecs.join(','),
                 Context: 'Streaming',
                 Protocol: 'http',
-                // If audio transcoding is needed, limit channels to number of physical audio channels
```
Comment (cvium) [inline src/scripts/browserDeviceProfile.js:731]: Why did you remove this?

Comment (nyanmisaka) [reply src/scripts/browserDeviceProfile.js:731]: `MaxAudioChannels: physicalAudioChannels.toString()`

This is already commonly used in most profiles above, so I think it's safe to remove the comment.

**Review by thornbill — APPROVED** [review]
